build.sbt

```diff
@@ -15,7 +15,7 @@ import munit.sbtmunit.BuildInfo.munitVersion
 import sbtcrossproject.CrossPlugin.autoImport.crossProject
 
 lazy val LanguageVersions = LatestScalaVersions
-lazy val LanguageVersion = LanguageVersions.head
+lazy val LanguageVersion = EarliestScala213
```

Comment:
This just make the projects that are not fully cross compiled to be by default earliest 2.13.x